client/src/modules/user/AuthBox/components/SignUpForm.jsx

```diff
@@ -110,35 +113,16 @@ class SignUpForm extends PureComponent {
       this.comparePasswords
     );
 
-  nameValidator = value => {
-    const { isLength } = validator;
+  nameValidator = value =>
```

Comment:
There are some similar functions that work in this way `xValidator = value => validator(value) ? '' : errorMessageId`. Consider this:
```javascript
validateName = validateWith(,
  value => validator.isLength(value, { min: 1, max: 32 })),
  'user.auth.input.email.invalid'
)
```
Where `validateWith = validator => errorMessageId => value => validator(value) ? '' : errorMessageId`. Then you can create a set of validation functions and validators for common fields that can be reused across the app.

Reply:
Added as a separated task to JIRA under EOC_408 task.